### PR TITLE
Fix for system cpu CLI output duplication 

### PR DESCRIPTION
**Pull request by arunsundark**
Test results:

~~~
admin@sonic:~$ curl -k -u admin:force10 https://localhost/restconf/data/openconfig-system:system/cpus/cpu=3
{"openconfig-system:cpu":[{"index":3,"state":{"idle":{"instant":65},"kernel":{"instant":21},"user":{"instant":10}}}]}
~~~

~~~
admin@sonic:~$ curl -k -u admin:force10 https://localhost/restconf/data/openconfig-system:system/cpus/
{"openconfig-system:cpus":{"cpu":[{"index":0,"state":{"idle":{"instant":62},"kernel":{"instant":26},"user":{"instant":9}}},{"index":1,"state":{"idle":{"instant":60},"kernel":{"instant":29},"user":{"instant":8}}},{"index":2,"state":{"idle":{"instant":59},"kernel":{"instant":30},"user":{"instant":8}}},{"index":3,"state":{"idle":{"instant":65},"kernel":{"instant":21},"user":{"instant":10}}},{"index":4,"state":{"idle":{"instant":64},"kernel":{"instant":23},"user":{"instant":10}}}]}}
~~~

### CODE REVIEW

File: src/translib/transformer/xfmr_system.go [hunk 1]
```diff
@@ -176,8 +175,8 @@ func getSystemCpu (idx int, cpu Cpu, syscpus *ocbinds.OpenconfigSystem_System_Cp
         log.Infof("syscpus.NewCpu failed")
         return
     }
-    ygot.BuildEmptyTree(syscpu)
-    syscpu.Index = &index
+   ygot.BuildEmptyTree(syscpu)
```
Comment (joyas-joseph) [inline src/translib/transformer/xfmr_system.go]: This to me is a workaround. Please find the actual issue.

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]
Same remark as Joyas. Holding merge

File: src/translib/transformer/xfmr_system.go (hunk 1 above)
Comment (arunsundark) [reply src/translib/transformer/xfmr_system.go]: Done

**Comment by arunsundark** [comment]
The actual issue is that, when a request comes to a leaf list, translib creates a new entry in the leaf list with the given key. In system app, another new entry is created in addition to the existing entry. This gives rise to duplicate entries resulting in duplication in the response.The latest commits check for the entry created by translib and avoids creating a new entry in case of a leaf list.

**Review by jeffyin-dell — APPROVED** [review]

**Comment by jeffyin-dell** [comment]
@arunsundark can you update the PR title/description in light of your latest revision?

**Comment by jeffyin-dell** [comment]
@joyas-joseph please review/merge

**Review by joyas-joseph — APPROVED** [review]

**Pull request merged**